module/zfs/spa.c

```diff
@@ -478,7 +478,7 @@ spa_prop_validate(spa_t *spa, nvlist_t *props)
 		case ZPOOL_PROP_LISTSNAPS:
 		case ZPOOL_PROP_AUTOEXPAND:
 			error = nvpair_value_uint64(elem, &intval);
-			if (!error && intval > 1)
+			if (!error && intval > ZIO_FAILURE_MODE_CONTINUE)
```

Comment:
These properties have nothing to do with the ZIO_FAILURE_MODE_* setting and using this macro here is misleading.  Best to leave this as is.